tests/test_kfactor.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+
+from pineko import kfactor
+
+
+class fakealpha:
```

Comment:
I know it is just for testing, but try to be consistent with naming conventions:
```suggestion
class FakeAlpha:
```

Reply:
[ee68474](https://github.com/NNPDF/pineko/pull/75/commits/ee684740e87309acb1659b98115ea7d5ea057ab9)